ExasTestClient/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using System.IO;
+using Exas;
+using Auth;
+
+namespace ExasTestClient
+{
+    class Program
+    {
+        private static ServiceProvider ServiceProvider;
+
+
+        static void Main(string[] args)
+        {
+            SetupDI();
+
+            var exasClient = ServiceProvider.GetService<IExasClient>();
+
+            var v = exasClient.GetCustomer("efba185a-ad2e-4863-abef-2838faf5cc16");
```

Comment:
Maybe we should consider removing the GUID here?

Reply:
Good spot, I removed it from the code